libraries/adaptive-expressions/src/builtinFunctions/empty.ts

```diff
@@ -37,9 +36,7 @@ export class Empty extends ComparisonEvaluator {
      */
     private static isEmpty(instance: any): boolean {
         let result: boolean;
-        if (instance === undefined) {
-            result = true;
-        } else if (typeof instance === 'string') {
+        if (typeof instance === 'string') {
```

Comment:
I think you still want to check that `instance != null` - `Object.keys` will throw for `null` or `undefined`.